spacewalk/setup/spacewalk-setup.spec

```diff
@@ -97,18 +97,14 @@ Requires:       perl-Satcon
 Requires:       spacewalk-admin
 Requires:       spacewalk-backend-tools
 Requires:       spacewalk-certs-tools
-%if 0%{?suse_version}
 %if 0%{?build_py3}
-Requires:       python3-PyYAML
+Requires:       (python3-PyYAML or python3-pyyaml)
 %else
-Requires:       python-PyYAML
+Requires:       (python-PyYAML or PyYAML)
 %endif
-%else
 %if 0%{?fedora} >= 22
 Recommends:     cobbler20
```

Comment:
Maybe it is time to drop this as well. Uyuni will not work anymore with cobbler20. We switched everything to cobbler 3.

Reply:
Can we keep this in for time being and put it into a cobbler 2 cleanup project? I fixed some cobbler code in #2883 which I also believe referenced some other files.

Reply:
sure, no problem